docs/source/logging/logging.md

```diff
@@ -1,54 +1,121 @@
+## How to perform logging in your Kedro project
+To add logging to your own code (e.g. in a node):
 
-By default, Python only shows logging messages at level `WARNING` and above. Kedro's logging configuration specifies that `INFO` level messages from Kedro should also be emitted. This makes it easier to track the progress of your pipeline when you perform a `kedro run`.
+```python
+import logging
 
-## Project-side logging configuration
+log = logging.getLogger(__name__)
+log.warning("Issue warning")
+log.info("Send information")
+log.debug("Useful information for debugging")
+```
 
-In addition to the `rich` handler defined in Kedro's framework, the [project-side `conf/base/logging.yml`](https://github.com/kedro-org/kedro/blob/main/kedro/templates/project/%7B%7B%20cookiecutter.repo_name%20%7D%7D/conf/base/logging.yml) defines two further logging handlers:
-* `console`: show logs on standard output (typically your terminal screen) without any rich formatting
-* `info_file_handler`: write logs of level `INFO` and above to `info.log`
+```{note}
+The name of a logger corresponds to a key in the `loggers` section of the logging configuration file (e.g. `kedro`). See [Python's logging documentation](https://docs.python.org/3/library/logging.html#logger-objects) for more information.
+```
 
-The logging handlers that are actually used by default are `rich` and `info_file_handler`.
+You can take advantage of Rich's [console markup](https://rich.readthedocs.io/en/stable/markup.html) in your logging calls:
 
-The project-side logging configuration also ensures that [logs emitted from your project's logger](#perform-logging-in-your-project) should be shown if they are `INFO` level or above (as opposed to the Python default of `WARNING`).
+```python
+log.error("[bold red blink]Important error message![/]", extra={"markup": True})
+```
 
-We now give some common examples of how you might like to change your project's logging configuration.
+## How to customise Kedro logging
 
 ### Using `KEDRO_LOGGING_CONFIG` environment variable
-
-`KEDRO_LOGGING_CONFIG` is an optional environment variable that you can use to specify the path of your logging configuration file, overriding the default Kedro's `default_logging.yml`.
-
-To use this environment variable, set it to the path of your desired logging configuration file before running any Kedro commands. For example, if you have a logging configuration file located at `/path/to/logging.yml`, you can set `KEDRO_LOGGING_CONFIG` as follows:
+To customise logging, specify the path of your logging configuration file by setting the environment variable `KEDRO_LOGGING_CONFIG` to override the default, which is Kedro's `default_logging.yml`. We recommend you put `logging.yml` inside the `conf` folder. For example, you can set `KEDRO_LOGGING_CONFIG` as follows:
```

Comment:
```suggestion
To customise logging in your Kedro project, you need to specify the path to a project-specific logging configuration file. Change the environment variable `KEDRO_LOGGING_CONFIG` to override the default logging configuration, which is Kedro's `default_logging.yml`. Point the variable instead to your project-specific configuration, which we recommend you name `logging.yml` and store inside the project's`conf` folder. 

For example, you can set `KEDRO_LOGGING_CONFIG` by typing the following into your terminal:
```

Reply:
Am I right in thinking that the Kedro project template gives you a `logging.yml` file by default? Maybe modify my suggestion to mention that, if it's true?